### PR TITLE
feat: migrate transactional emails to cio

### PR DESCRIPTION
**Pull request by idoshamun**
Still WIP but most of the work is done. I placed some todos that I need to fix, I'm waiting for a response from their support.

It actually required more changes than I expected as some things are not supported:
* Sending emails in batches with personalization parameters
* Generating batch id for the digest

### CODE REVIEW

File: __tests__/notifications/index.ts [hunk 1]
```diff
@@ -1049,24 +991,6 @@ describe('storeNotificationBundle', () => {
     expect(actual.attachments.length).toEqual(0);
   });
 
-  it('should generate squad_access notification', () => {
```
Comment (idoshamun) [inline __tests__/notifications/index.ts:1052]: This notification is no longer used

File: src/growthbook.ts [hunk 2]
```diff
@@ -127,7 +119,6 @@ export const features = {
   }),
   dailyDigest: new Feature('daily_personalized_digest', {
     ...digestFeatureBaseConfig,
-    templateId: 'd-925d2759ddd641f99220b3c7c6836458',
```
Comment (capJavert) [inline src/growthbook.ts:130]: Digests now have the same template ID by default?

File: src/routes/index.ts [hunk 3]
```diff
@@ -131,4 +136,32 @@ Disallow: /`);
       message: 'ok',
     });
   });
+
+  // https://customer.io/docs/journeys/custom-unsubscribe-links/#step-1-configure-your-system-to-receive-the-post-when-recipients-click-unsubscribe
+  fastify.post<{ Querystring: { token: string } }>(
+    '/unsubscribe',
```
Comment (capJavert) [inline src/routes/index.ts:142]: Maybe move under `/notifications`, separate file or just add to path.

**Review by capJavert — COMMENTED** [review]
Minor stuff so far, actually surprised because I expected more major changes. Nice work 🚀

File: src/growthbook.ts [hunk 4]
```diff
@@ -104,15 +104,7 @@ export class Feature<T extends JSONValue> {
...
-  templateId: 'd-328d1104d2e04fa1ab91e410e02751cb',
-  meta: {
-    from: {
-      email: 'informer@daily.dev',
-      name: 'daily.dev',
-    },
-    category: 'Digests',
-    asmGroupId: 23809,
-  },
+  templateId: '46',
```
Comment (rebelchris) [inline src/growthbook.ts:107]: Don't we need to update more templateId's?
Or we'll do the main ones seperate later?

**Review by rebelchris — APPROVED** [review]
Wow really not that bad of a change!

File: src/growthbook.ts (hunk 2 above)
Comment (idoshamun) [reply src/growthbook.ts:130]: yes. nimrod asked to merge the templates together

**Pull request merged**